README.md

```diff
@@ -255,6 +255,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
   - [Mercurial](http://mercurial.selenic.com/)
   - [MercuryMover](http://www.heliumfoot.com/mercurymover/)
   - [Messages](http://www.apple.com/osx/apps/#messages)
+  - [Microsoft Lync](http://office.microsoft.com/lync)
```

Comment:
Ah… Microsoft is re-branding Lync as Skype for Business.  But, I didn’t realize that it wouldn’t provide a download link unless one’s microsoft account was tied to a business that used Lync.  I think this particular entry would be of extremely limited use, so it probably wouldn’t make sense to add it to the master.

On Apr 20, 2015, at 11:15 PM, Laurent Raufaste <notifications@github.com<mailto:notifications@github.com>> wrote:


In README.md<https://github.com/lra/mackup/pull/523#discussion_r28747339>:

> @@ -255,6 +255,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
>    - [Mercurial](http://mercurial.selenic.com/)
>    - [MercuryMover](http://www.heliumfoot.com/mercurymover/)
>    - [Messages](http://www.apple.com/osx/apps/#messages)
> +  - [Microsoft Lync](http://office.microsoft.com/lync)


The link is not working for me, it redirects me to some skype page =(

—
Reply to this email directly or view it on GitHub<https://github.com/lra/mackup/pull/523/files#r28747339>.